equipment-datastore-common-test/src/main/java/com/telecominfraproject/wlan/equipment/datastore/BaseEquipmentDatastoreTest.java

```diff
@@ -348,6 +348,141 @@ public void testEquipmentPagination()
+       page4.getItems().forEach(e -> assertEquals(customerId_1, e.getCustomerId()) );
+       page5.getItems().forEach(e -> assertEquals(customerId_1, e.getCustomerId()) );
+       
+       assertEquals(0, page6.getItems().size());
+       assertEquals(0, page7.getItems().size());
+       
+       assertFalse(page1.getContext().isLastPage());
+       assertFalse(page2.getContext().isLastPage());
+       assertFalse(page3.getContext().isLastPage());
+       assertFalse(page4.getContext().isLastPage());
+       assertFalse(page5.getContext().isLastPage());
+       
+       assertTrue(page6.getContext().isLastPage());
+       assertTrue(page7.getContext().isLastPage());
+       
+       // Check search on name
+       PaginationContext<Equipment> context_name = new PaginationContext<>(10);
+       PaginationResponse<Equipment> page1_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, context_name);
+       PaginationResponse<Equipment> page2_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page1_name.getContext());
+       PaginationResponse<Equipment> page3_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page2_name.getContext());
+       PaginationResponse<Equipment> page4_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page3_name.getContext());
+       PaginationResponse<Equipment> page5_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page4_name.getContext());
+       PaginationResponse<Equipment> page6_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page5_name.getContext());
+       PaginationResponse<Equipment> page7_name = testInterface.searchByMacAndName(customerId_1, "qr", sortBy, page6_name.getContext());
+       
+       //verify search on name and mac returned the same items
+       assertEquals(page1.getItems(), page1_name.getItems());
+       assertEquals(page2.getItems(), page2_name.getItems());
+       assertEquals(page3.getItems(), page3_name.getItems());
+       assertEquals(page4.getItems(), page4_name.getItems());
+       assertEquals(page5.getItems(), page5_name.getItems());
+       assertEquals(page6.getItems(), page6_name.getItems());
+       assertEquals(page7.getItems(), page7_name.getItems());
+       
+       List<String> expectedPage3Strings = new ArrayList<>(Arrays.asList(new String[]{"qr_27", "qr_28", "qr_29", "qr_3", "qr_30", "qr_31", "qr_32", "qr_33", "qr_34", "qr_35" }));
+       List<String> actualPage3Strings = new ArrayList<>();
+       page3.getItems().stream().forEach( ce -> actualPage3Strings.add(ce.getName()) );
+       
+       assertEquals(expectedPage3Strings, actualPage3Strings);
+       
+       //test first page of the results with empty sort order -> default sort order (by Id ascending)
+       PaginationResponse<Equipment> page1EmptySort = testInterface.getForCustomer(customerId_1, Collections.emptyList(), context);
```

Comment:
should test testInterface.searchByMacAndName, not testInterface.getForCustomer in here